### PR TITLE
Helm chart: remove mentions of legacy storage types

### PR DESCRIPTION
**Pull request by adutra**

### CODE REVIEW

**Review by ajantha-bhat — DISMISSED** [review]
LGTM. It is as per the new model.

File: helm/nessie/values.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # -- The number of replicas to deploy (horizontal scaling).
-# Beware that replicas are stateless; don't set this number > 1 when using INMEMORY or ROCKS version store types.
+# Beware that replicas are stateless; don't set this number > 1 when using IN_MEMORY or ROCKSDB version store types.
```
Comment (ajantha-bhat) [inline helm/nessie/values.yaml:2]: Line 31 and many places in this file still uses the old model names. Do we need to replace them too?

**Review by dimas-b — APPROVED** [review]

File: helm/nessie/values.yaml (hunk 1 above)
Comment (adutra) [reply helm/nessie/values.yaml:2]: Good catch, I will do a more thorough pass on the file.

**Review by dimas-b — APPROVED** [review]
I guess we need to make the PR title more generic now :)

**Pull request merged**